crates/noirc_frontend/src/lexer/token.rs

```diff
@@ -325,7 +325,8 @@ pub enum Attribute {
     Builtin(String),
     Oracle(String),
     Deprecated(Option<String>),
-    Test,
+    // Boolean flag that represents if the test should fail
+    Test(bool),
```

Comment:
```suggestion
    Test { expect_failure: bool },
```
Don't use comments when you can communicate purpose via the code.

Reply:
Alternatively, we can rework this to be two variants
```suggestion
    TestSuccess,
    TestFailure,
```